proxy/dkg.go

```diff
@@ -162,6 +181,7 @@ func (d dkg) EditDKGActor(w http.ResponseWriter, r *http.Request) {
 				dela.Logger.Err(err).Msg("failed to setup")
 			}
 		}()
+		// begin the decryption
```

Comment:
```suggestion
	// begin the decryption
```

just for consistency with the case setup